spin-up-hub/src/store.js

```diff
@@ -15,7 +15,7 @@ const store = createStore({
         tags: [], language: "", createdAt: "", lastUpdated: "", spinVersion: "",
         artifactSource: ""
       },
-      contentTypes: ["Plugin", "Template", "Library", "Sample", "Component"],
+      contentTypes: ["Plugin", "Template", "Library", "Sample", "Component", "Pattern", "Architecture"],
```

Comment:
What's the difference between a pattern and an architecture in this context?

Reply:
IMO "patterns" are smaller things. E.g. When you build an HTTP API (architecture), you may use CORS or Content Negotiation (pattern)